cmd/kube-apiserver/app/server.go

```diff
@@ -51,7 +51,8 @@ import (
 const (
 	// Maximum duration before timing out read/write requests
 	// Set to a value larger than the timeouts in each watch server.
-	ReadWriteTimeout = time.Minute * 60
+	ReadWriteTimeout            = time.Minute * 60
+	defaultLongRunningRequestRE = "(/watch(/.*|)$|/proxy(/.*|)$|logs(/|)$|portforward(/|)$|exec(/|)$)"
```

Comment:
also add a TODO that this still needs tightening (currently matches path segments for namespaces/names of things named "watch", "proxy", etc